<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QScrollArea" name="responseArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>631</width>
     <height>271</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>629</width>
      <height>269</height>
     </rect>
    </property>
    <widget class="QTextBrowser" name="responseViewer">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>631</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QScrollArea" name="requestArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>109</y>
     <width>631</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noteworthy</family>
    </font>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>629</width>
      <height>179</height>
     </rect>
    </property>
    <widget class="QTextEdit" name="requestEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>631</width>
       <height>181</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
      </font>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="sendBtn">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>52</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Phosphate</family>
    </font>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="uriEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>591</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>67</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>URI:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
